packages/cli/generation/local-generation/local-workspace-runner/src/runGenerator.ts

```diff
@@ -157,7 +157,7 @@ async function writeIrToFile({
         tmpdir: workspaceTempDir.path
     });
     const absolutePathToIr = AbsoluteFilePath.of(irFile.path);
-    await streamObjectToFile(absolutePathToIr, ir, { pretty: true });
+    await streamObjectToFile(absolutePathToIr, ir, { pretty: false });
```

Comment:
This reduces the file size of the generated IR JSON significantly. Makes it a bit more tedious to debug, but running a prettifier on the file solves that problem in a jiffy.